article/app/controllers/LiveBlogController.scala

```diff
@@ -366,6 +366,6 @@ object LiveBlogController {
   }
 
   def checkIfSupported(blog: PageWithStoryPackage): Boolean = {
-    isDeadBlog(blog) && isSupportedTheme(blog) && isNotRecent(blog)
+    isDeadBlog(blog) && isSupportedTheme(blog)
```

Comment:
Can the definition for `isNotRecent` be deleted as well?

Reply:
I opted to keep it because we are sending this data to our log store and it could be useful to continue to record this. Once we go full DCR we will remove the entire `dcrCouldRender` thing